vue.config.js

```diff
@@ -9,6 +9,12 @@ const commitHash = require('child_process')
 
 module.exports = {
   lintOnSave: false,
+  pluginOptions: {
+    ssr: {
+      favicon: './dist/assets/favicon.ico',
+      nodeExternalsWhitelist: [/\.css$/, /\?vue&type=style/, 'lodash-es'],
```

Comment:
This is based on https://github.com/Akryum/vue-cli-plugin-ssr/issues/45#issuecomment-452719512 and fixes the problem with `lodash-es`, but it doesn't solve the same problem with our SDK.